interface/billing/ub04_dispose.php

```diff
@@ -0,0 +1,432 @@
+<?php
+/**
+ * UB04 Functions
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
+ */
+require_once("../globals.php");
+require_once("$srcdir/Claim.class.php");
+require_once("$srcdir/pdf_creator.class.php");
```

Comment:
Also. Almost forgot. Since removing the Claim.class.php include, need to add the following:
``````
require_once("$srcdir/invoice_summary.inc.php");
``````